react-app/src/components/PatientData.js

```diff
@@ -5,10 +5,10 @@ import LogList from './LogList';
 function PatientData(props) {
   return (
     <div>
-      {props.id === null && !props.showLogs && (
```

Comment:
I think we still want to have the `!props.showLogs` condition in this statement, right? I'm guessing it just got lost in a rebase.